ci/centos-7/Dockerfile

```diff
@@ -38,11 +44,21 @@ RUN yum -y install \
     which \
   && yum clean all && rm -rf /var/cache/yum
 
+# Install a recent flex for Spicy.
+# ADD https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz /tmp/flex
+# RUN curl -sSL "https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-x86_64.tar.gz" | tar xzf - -C "${CMAKE_DIR}" --strip-components 1 \
+RUN curl -sSL "https://github.com/westes/flex/releases/download/v${FLEX_VERSION}/flex-${FLEX_VERSION}.tar.gz" | tar xzf - -C /tmp \
+ && (cd /tmp/flex-${FLEX_VERSION} \
+     && ./configure --prefix=${FLEX_DIR} \
+     && make -j`nproc` install) \
+ && rm -rf /tmp/flex-${FLEX_VERSION}
+
```

Comment:
Spicy heavily uses object libraries (both to propagate flags as well as link dependencies). Support for that is incomplete with the CMake older than 3.15; Zeek OTOH currently requires CMake from the range 3.5...3.18.